narrow_valley.py

```diff
@@ -0,0 +1,647 @@
+import time
+import random
+
+#<------------------------------------------------------------ Basic Functions ------------------------------------------------------------>    
+
+def coin_flip():
+	return random.choice(['heads', 'tails'])
+
+def die_roll():
+	return random.randint(1, 6)
+
+def time_print(string):
+    print(string)
+    time.sleep(2)
+
+def time_print_loop(lst):
+    for i in range(len(lst)):
+        time_print(lst[i])
+
+def time_print_img(lst):
+    for i in range(len(lst)):
+        print(lst[i])
```

Comment:
Instead of looping through `lst` by using for `i in range`..., you can use the element directly: `for el in lst`